src/plots/cartesian/set_convert.js

```diff
@@ -417,6 +417,10 @@ module.exports = function setConvert(ax) {
     ax._min = [];
     ax._max = [];
 
+    // copy ref to fullLayout.separators so that
+    // methods in Axes can use it w/o having to pass fullLayout
+    ax.separators = fullLayout.separators;
```

Comment:
shall we call it `_separators` to distinguish it from regular settings managed by `supplyDefaults`? That actually adds overhead (as `relinkPrivateKeys` will look at it) but it does seem to be the pattern we've been using.